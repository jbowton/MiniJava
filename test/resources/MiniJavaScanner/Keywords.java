class
public
static
void
main
String
extends
return
int
boolean
if
else
while
System.out.println
length
false
this
new