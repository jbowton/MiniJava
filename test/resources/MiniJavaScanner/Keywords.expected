CLASS PUBLIC STATIC VOID MAIN STRING EXTENDS RETURN INT BOOLEAN IF ELSE WHILE PRINTLN LENGTH FALSE THIS NEW 